### ISSUE TITLE
playwright runner. Execute specs against rest api without raising browser?

### ISSUE THREAD

**Issue by vicentlluis** [microsoft/playwright#16587]
I would line to know is it possible to run for example 
a project that contains specs that don't require any browser to be raised?

For example, let's imagine a project that only have specs to test a rest api, and I need to run with Playwright runner?

Is is possible? It would be a very good advantage

**Comment by aslushnikov** [comment]
> Is is possible? It would be a very good advantage

@vicentlluis yes it is. If you don't use `page`, `browser` or `context` fixtures, then browser will not be instantiated. 

For example, you can do pure API tests with just the `request` fixture:

```js
test('should query data', async ({ request }) => {
  const response = await request.get('https://example.com');
  // ...
});
